src/ts/components/transaction/Fee.tsx

```diff
@@ -74,7 +108,7 @@ class Fee extends React.Component<IFeeProps, IFeeState> {
   renderBalance () {
```

Comment:
perhaps rename `renderBalance` to `renderFee` to avoid confusion

Reply:
done